.gitignore

```diff
@@ -0,0 +1,3 @@
+cloudevents/go/cde/build/
```

Comment:
What does `cde` stand for?

Reply:
What about `sdk/go/` or `cde/sdk/go`?
If we are going to provide an sdk in other languages with would then have:
```
cde/
  \-sdk
      |- go
      |- python
      \- rust
```

Reply:
CD like CD Foundation and E for Events.. I am terrible at naming.. but totally up for that package structure..

Reply:
but you are missing the most important one "java" :)

Reply:
I'm totally going to write a Java binding :P

Reply:
> CD like CD Foundation and E for Events.. I am terrible at naming.. but totally up for that package structure..

cde is actually nice :)
And naming is one of the most difficult things to do!

Can we add support for AB testing events in the CDE protocol? 
I'd love ABCDE events :D

Reply:
amazing best cd events :D

Reply:
@afrittoli refactored to use the cde/sdk/go path..

Reply:
I will resolve this one.. as the new structure was added